### PR TITLE
🌐 Add Ukrainian translation for `docs/uk/docs/fastapi-people.md`, `docs/uk/docs/python-types.md`, `docs/uk/docs/tutorial/index.md`

### PR DESCRIPTION
**Pull request by ss-o-furda**
Missed documentation on `docs/uk/docs/external-links.md` because it is translated into #4037 and `docs/uk/docs/features.md`/`docs/uk/docs/index.md` as they are translated into #1791

### CODE REVIEW

**Comment by tiangolo** [comment]
Thanks for your interest!

It's best to have each translation file as a separate PR, that way each one can be independently reviewed, and as soon one of the translations is ready it can be merged. But with all of them on the same PR, they would all have to be ready and perfect for the whole PR to be merged. And also, reviewers would have to be able to make time to review all the changed files before approving the whole PR, so it would take a lot longer. So, please separate the translations into separate PRs. :pray: 

Thanks!

**Comment by tiangolo** [comment]
Maybe you or others can take parts/files of this PR and make them as independent PRs. That would work. :nerd_face:

File: docs/uk/docs/python-types.md [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+
+Це дуже проста програма.
+
+Але тепер уявіть, що ви писали це з нуля.
+
+У певний момент ви розпочали б оголошення функції, у вас були б готові параметри...
+
+Але тоді вам потрібно викликати "той метод який переводить першу літеру у верхній регістр".
+
+Це було `upper`? Чи `uppercase`? `first_uppercase`? `capitalize`?
```
Comment (lizaliza123) [inline docs/uk/docs/python-types.md:52]: Це буде `upper`? Чи `uppercase`? `first_uppercase`? `capitalize`?

File: docs/uk/docs/python-types.md [hunk 2]
```diff
@@ -0,0 +1,448 @@
...
+
+Тоді ви спробуєте з давнім другом програміста - автозаповненням редактора.
```
Comment (lizaliza123) [inline docs/uk/docs/python-types.md:54]: Тоді ви спробуєте давнього друга програміста - автозаповнення редактора.

File: docs/uk/docs/python-types.md [hunk 3]
```diff
@@ -0,0 +1,448 @@
...
+{!../../../docs_src/python_types/tutorial005.py!}
+```
+
+### Generic-типи з параметрами типів
+
+Існують деякі структури даних, які можуть містити інші значення, наприклад `dict`, `list`, `set` та `tuple`. І внутрішні значення також можуть мати свій тип.
+
+Ці типи, які мають внутрішні типи, називаються "**generic**" типами. І оголосити їх можна навіть із внутрішніми типами.
+
+Щоб оголосити ці типи та внутрішні типи, ви можете використовувати стандартний модуль Python `typing`. Він існує спеціально для підтримки цих анотацій типів.
```
Comment (lizaliza123) [inline docs/uk/docs/python-types.md:153]: Щоб оголосити ці типи та внутрішні типи, ви можете використовувати стандартний модуль Python `typing`. Він існує спеціально для підтримки анотацій типів.

File: docs/uk/docs/python-types.md [hunk 4]
```diff
@@ -0,0 +1,448 @@
...
+Це означає:
+
+* Змінна `items_t` це `tuple` з 3 елементами, `int`, ще `int`, та `str`.
+* Змінна `items_s` це `set`, і кожен його елемент типу `bytes`.
+
+#### Dict (словник)
+
+Щоб оголосити `dict`, вам потрібно передати 2 параметри типу, розділені комами.
+
+Перший параметр типу для ключа у `dict`.
```
Comment (lizaliza123) [inline docs/uk/docs/python-types.md:242]: Перший параметр типу для ключа `dict`.

File: docs/uk/docs/python-types.md [hunk 5]
```diff
@@ -0,0 +1,448 @@
...
+
+Другий параметр типу для значення у `dict`:
```
Comment (lizaliza123) [inline docs/uk/docs/python-types.md:244]: Другий параметр типу для значення `dict`:

File: docs/uk/docs/python-types.md [hunk 6]
```diff
@@ -0,0 +1,448 @@
...
+
+Це означає:
+
+* Змінна `prices` це `dict`:
+    * Ключі цього `dict` типу `str` (наприклад, назва кожного елементу).
+    * Значення цього `dict` типу `float` (наприклад, ціна кожного елементу).
+
+#### Union (об'єднання)
+
+Ви можете оголосити, що змінна може бути будь-яким із **кількох типів**, наприклад, `int` або `str`.
```
Comment (lizaliza123) [inline docs/uk/docs/python-types.md:266]:
```suggestion
Ви можете оголосити, що змінна є будь-яким із **кількох типів**, наприклад, `int` або `str`.
```

File: docs/uk/docs/python-types.md [hunk 7]
```diff
@@ -0,0 +1,448 @@
...
+    * `Optional` (так само як у Python 3.6)
+    * ...та інші.
+
+    У Python 3.10, як альтернатива використанню `Union` та `Optional`, ви можете використовувати <abbr title='також називають «побітовим "або" оператором», але це значення тут не актуальне'>вертикальну смугу (`|`)</abbr> щоб оголосити об'єднання типів.
+
+### Класи як типи
+
+Ви також можете оголосити клас як тип змінної.
+
+Скажімо, у вас є клас `Person` з іменем:
```
Comment (lizaliza123) [inline docs/uk/docs/python-types.md:370]:
```suggestion
Скажімо, у вас є клас `Person` з ім'ям:
```

File: docs/uk/docs/python-types.md [hunk 8]
```diff
@@ -0,0 +1,448 @@
...
+
+## Pydantic моделі
+
+<a href="https://pydantic-docs.helpmanual.io/" class="external-link" target="_blank">Pydantic</a> це бібліотека Python для валідації даних.
+
+Ви оголошуєте «форму» даних як класи з атрибутами.
+
+І кожен атрибут має тип.
+
+Потім ви створюєте екземпляр цього класу з деякими значеннями, і він перевірить значення, перетворить їх у відповідний тип (якщо є потреба) і надасть вам об’єкт з усіма даними.
```
Comment (lizaliza123) [inline docs/uk/docs/python-types.md:394]:
```suggestion
Потім ви створюєте екземпляр цього класу з деякими значеннями, і він перевірить ці значення, перетворить їх у відповідний тип (якщо є потреба) і надасть вам об’єкт з усіма даними.
```

File: docs/uk/docs/python-types.md [hunk 9]
```diff
@@ -0,0 +1,448 @@
...
+
+**FastAPI** повністю базується на Pydantic.
+
+Ви побачите набагато більше цього всього на практиці в [Tutorial - User Guide](tutorial/index.md){.internal-link target=_blank}.
+
+## Анотації типів у **FastAPI**
+
+**FastAPI** використовує ці підказки для виконання кількох речей.
+
+З **FastAPI** ви оголошуєте параметри з підказками типу, і ви отримуєте:
```
Comment (lizaliza123) [inline docs/uk/docs/python-types.md:429]:
```suggestion
З **FastAPI** ви оголошуєте параметри з підказками типу, і отримуєте:
```

File: docs/uk/docs/python-types.md [hunk 10]
```diff
@@ -0,0 +1,448 @@
...
+    * Генерування **автоматичних помилок** що повертаються клієнту, коли дані недійсні.
+* **Документування** API за допомогою OpenAPI:
+    * який потім використовується для автоматичної інтерактивної документації користувальницьких інтерфейсів.
+
+Все це може здатися абстрактним. Не хвилюйся. Ви побачите все це в дії в [Туторіал - Посібник користувача](tutorial/index.md){.internal-link target=_blank}.
+
+Важливо те, що за допомогою стандартних типів Python в одному місці (замість того, щоб додавати більше класів, декораторів тощо), **FastAPI** зробить багато роботи за вас.
+
+!!! info
+    Якщо ви вже пройшли весь навчальний посібник і повернулися, щоб дізнатися більше про типи, хороший ресурс це <a href="https://mypy.readthedocs.io/en/latest/cheat_sheet_py3.html" class="external-link" target="_blank">"шпаргалка" від `mypy`</a>.
```
Comment (lizaliza123) [inline docs/uk/docs/python-types.md:448]:
```suggestion
    Якщо ви вже пройшли весь навчальний посібник і повернулися, щоб дізнатися більше про типи, ось хороший ресурс <a href="https://mypy.readthedocs.io/en/latest/cheat_sheet_py3.html" class="external-link" target="_blank">"шпаргалка" від `mypy`</a>.
```

**Review by lizaliza123 — COMMENTED** [review]
Thank you for translation

File: docs/uk/docs/fastapi-people.md [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+</div>
+{% endif %}
+
+Я є творцем і супроводжувачем **FastAPI**. Детальніше про це можна прочитати в [Довідка FastAPI - Отримати довідку - Зв'язатися з автором](help-fastapi.md#connect-with-the-author){.internal-link target=_blank}.
+
+...Але тут я хочу показати вам спільноту.
+
+---
+
+**FastAPI** отримує велику підтримку від спільноти. І я хочу виділити їхній внесок.
```
Comment (romashevchenko) [inline docs/uk/docs/fastapi-people.md:27]:
```suggestion
**FastAPI** отримує велику підтримку від спільноти. І я хочу відзначити їхній внесок.
```

File: docs/uk/docs/fastapi-people.md [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+
+* [Допомагають іншим із проблемами (запитаннями) у GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank}.
+* [Створюють пул реквести](help-fastapi.md#create-a-pull-request){.internal-link target=_blank}.
+* Переглядають пул реквести, [особливо важливо для перекладів](contributing.md#translations){.internal-link target=_blank}.
+
+Оплески їм. 👏 🙇
+
+## Найбільш активні користувачі минулого місяця
+
+Це користувачі, які [найбільш допомагали іншим із проблемами (запитаннями) у GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank} протягом минулого місяця. ☕
```
Comment (romashevchenko) [inline docs/uk/docs/fastapi-people.md:39]:
```suggestion
Це користувачі, які [найбільше допомагали іншим із проблемами (запитаннями) у GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank} протягом минулого місяця. ☕
```

File: docs/uk/docs/fastapi-people.md [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Експерти
+
+Ось **експерти FastAPI**. 🤓
+
+Це користувачі, які [найбільш допомагали іншим із проблемами (запитаннями) у GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank} протягом *всього часу*.
```
Comment (romashevchenko) [inline docs/uk/docs/fastapi-people.md:55]:
```suggestion
Це користувачі, які [найбільше допомагали іншим із проблемами (запитаннями) у GitHub](help-fastapi.md#help-others-with-issues-in-github){.internal-link target=_blank} протягом *всього часу*.
```

File: docs/uk/docs/fastapi-people.md [hunk 14]
```diff
@@ -0,0 +1,178 @@
...
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Issues replied: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Найкращі учасники
+
+Ось **Найкращі учасники**. 👷
```
Comment (romashevchenko) [inline docs/uk/docs/fastapi-people.md:71]:
```suggestion
Ось **Найкращі співавтори**. 👷
```

File: docs/uk/docs/fastapi-people.md [hunk 15]
```diff
@@ -0,0 +1,178 @@
...
+<div class="user-list user-list-center">
+{% for user in people.experts %}
+
+<div class="user"><a href="{{ user.url }}" target="_blank"><div class="avatar-wrapper"><img src="{{ user.avatarUrl }}"/></div><div class="title">@{{ user.login }}</div></a> <div class="count">Issues replied: {{ user.count }}</div></div>
+{% endfor %}
+
+</div>
+{% endif %}
+
+## Найкращі учасники
```
Comment (romashevchenko) [inline docs/uk/docs/fastapi-people.md:69]:
```suggestion
## Найкращі співавтори
```

File: docs/uk/docs/python-types.md [hunk 16]
```diff
@@ -0,0 +1,448 @@
+# Вступ до типів Python
+
+Python підтримує додаткові "type hints".
```
Comment (romashevchenko) [inline docs/uk/docs/python-types.md:3]:
```suggestion
Python підтримує додаткові "підказки типу" ("type hints") (також звані "анотаціями типу" ("type annotations")).

File: docs/uk/docs/python-types.md [hunk 17]
```diff
@@ -0,0 +1,448 @@
...
+
+Ці **"type hints"** є спеціальним синтаксисом що дозволяє оголошувати <abbr title="наприклад: str, int, float, bool">тип</abbr> змінної.
```
Comment (romashevchenko) [inline docs/uk/docs/python-types.md:5]:
```suggestion
Ці **"type hints"** є спеціальним синтаксисом, що дозволяє оголошувати <abbr title="наприклад: str, int, float, bool">тип</abbr> змінної.
```

File: docs/uk/docs/python-types.md [hunk 18]
```diff
@@ -0,0 +1,448 @@
...
+
+За допомогою оголошення типів для ваших змінних, редактори та інструменти можуть надати вам кращу підтримку.
+
+Це просто **швидкий посібник / нагадування** про анотації типів у Python. Він покриває лише мінімум необхідний щоб використовувати їх з **FastAPI**... що насправді дуже мало.
```
Comment (romashevchenko) [inline docs/uk/docs/python-types.md:9]:
```suggestion
Це просто **швидкий посібник / нагадування** про анотації типів у Python. Він покриває лише мінімум, необхідний щоб використовувати їх з **FastAPI**... що насправді дуже мало.
```

File: docs/uk/docs/python-types.md [hunk 19]
```diff
@@ -0,0 +1,448 @@
...
+{!../../../docs_src/python_types/tutorial001.py!}
+```
+
+### Редагуйте це
+
+Це дуже проста програма.
+
+Але тепер уявіть, що ви писали це з нуля.
+
+У певний момент ви розпочали б оголошення функції, у вас були б готові параметри...
```
Comment (romashevchenko) [inline docs/uk/docs/python-types.md:48]:
```suggestion
У певний момент ви розпочали б визначення функції, у вас були б готові параметри...
```

File: docs/uk/docs/python-types.md [hunk 20]
```diff
@@ -0,0 +1,448 @@
...
+
+Але тоді вам потрібно викликати "той метод який переводить першу літеру у верхній регістр".
```
Comment (romashevchenko) [inline docs/uk/docs/python-types.md:50]:
```suggestion
Але тоді вам потрібно викликати "той метод, який переводить першу літеру у верхній регістр".
```

File: docs/uk/docs/python-types.md [hunk 21]
```diff
@@ -0,0 +1,448 @@
...
+
+Це було `upper`? Чи `uppercase`? `first_uppercase`? `capitalize`?
+
+Тоді ви спробуєте з давнім другом програміста - автозаповненням редактора.
+
+Ви надрукуєте перший параметр функції, `first_name`, тоді крапку (`.`), а тоді натиснете `Ctrl+Space` щоб запустити автозаповнення.
```
Comment (romashevchenko) [inline docs/uk/docs/python-types.md:56]:
```suggestion
Ви надрукуєте перший параметр функції, `first_name`, тоді крапку (`.`), а тоді натиснете `Ctrl+Space`, щоб запустити автозаповнення.
```

File: docs/uk/docs/python-types.md [hunk 22]
```diff
@@ -0,0 +1,448 @@
...
+
+```Python hl_lines="1"
+{!../../../docs_src/python_types/tutorial003.py!}
+```
+
+Оскільки редактор знає типи змінних, ви не тільки отримаєте автозаповнення, ви також отримаєте перевірку помилок:
+
+<img src="/img/python-types/image04.png">
+
+Тепер ви знаєте, щоб виправити це вам потрібно перетворити `age` у строку з допомогою `str(age)`:
```
Comment (romashevchenko) [inline docs/uk/docs/python-types.md:120]:
```suggestion
Тепер ви знаєте, щоб виправити це, вам потрібно перетворити `age` у строку з допомогою `str(age)`:
```

File: docs/uk/docs/python-types.md [hunk 23]
```diff
@@ -0,0 +1,448 @@
...
+
+    З модуля `typing`, імпортуємо `List` (з великої літери `L`):
+
+    ``` Python hl_lines="1"
+    {!> ../../../docs_src/python_types/tutorial006.py!}
+    ```
+
+    Оголосимо змінну з тим самим синтаксисом двокрапки (`:`).
+
+    Як тип вкажемо `List` який ви імпортували з `typing`.
```
Comment (romashevchenko) [inline docs/uk/docs/python-types.md:177]:
```suggestion
    Як тип вкажемо `List`, який ви імпортували з `typing`.
```

File: docs/uk/docs/python-types.md [hunk 24]
```diff
@@ -0,0 +1,448 @@
...
+
+<img src="/img/python-types/image05.png">
+
+Без типів цього майже неможливо досягти.
+
+Зверніть увагу, що змінна `item` є одним із елементів у списку `items`.
+
+І все ж редактор знає, що це `str`, і надає підтримку для цього.
+
+#### Tuple and Set (кортеж та набір)
```
Comment (romashevchenko) [inline docs/uk/docs/python-types.md:217]:
```suggestion
#### Tuple and Set (Кортеж та набір)
```

File: docs/uk/docs/python-types.md [hunk 25]
```diff
@@ -0,0 +1,448 @@
...
+    {!> ../../../docs_src/python_types/tutorial008b.py!}
+    ```
+
+=== "Python 3.10 і вище"
+
+    ```Python hl_lines="1"
+    {!> ../../../docs_src/python_types/tutorial008b_py310.py!}
+    ```
+
+В обох випадках це означає що `item` може бути `int` або `str`.
```
Comment (romashevchenko) [inline docs/uk/docs/python-types.md:284]:
```suggestion
В обох випадках це означає, що `item` може бути `int` або `str`.
```

File: docs/uk/docs/tutorial/index.md [hunk 26]
```diff
@@ -0,0 +1,80 @@
...
+
+## Розширений посібник користувача
+
+Існує також **Розширений посібник користувача**, який ви зможете прочитати пізніше після цього **Туторіал - Посібник користувача**.
+
+**Розширений посібник користувача** засновано на цьому, використовує ті самі концепції та навчає вас деяким додатковим функціям.
+
+Але вам слід спочатку прочитати **Туторіал - Посібник користувача** (те, що ви зараз читаєте).
+
+Він розроблений таким чином, що ви можете створити повну програму лише за допомогою **Туторіал - Посібник користувача**, а потім розширити його різними способами, залежно від ваших потреб, використовуючи деякі з додаткових ідей з **Розширеного посібника користувача** .
```
Comment (romashevchenko) [inline docs/uk/docs/tutorial/index.md:80]:
```suggestion
Він розроблений таким чином, що ви можете створити повну програму лише за допомогою **Туторіал - Посібник користувача**, а потім розширити її різними способами, залежно від ваших потреб, використовуючи деякі з додаткових ідей з **Розширеного посібника користувача** .
```

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**